annotation.go

```diff
@@ -81,11 +81,14 @@ type AnnotationService interface {
 	// ListStreams lists all streams matching the filter.
 	ListStreams(ctx context.Context, orgID platform.ID, filter StreamListFilter) ([]ReadStream, error)
 	// CreateOrUpdateStream creates or updates the matching stream by name.
-	CreateOrUpdateStream(ctx context.Context, stream Stream) (*ReadStream, error)
+	CreateOrUpdateStream(ctx context.Context, orgID platform.ID, stream Stream) (*ReadStream, error)
+	// GetStream gets a stream by id. Currently this is only used for authorization, and there are no
+	// API routes for getting a single stream by ID.
+	GetStream(ctx context.Context, id platform.ID) (*StoredStream, error)
 	// UpdateStream updates the stream by the ID.
 	UpdateStream(ctx context.Context, id platform.ID, stream Stream) (*ReadStream, error)
 	// DeleteStreams deletes one or more streams by name.
-	DeleteStreams(ctx context.Context, delete BasicStream) error
+	DeleteStreams(ctx context.Context, orgID platform.ID, delete BasicStream) error
```

Comment:
Needs the `orgID` here as well since the backend needs to know which orgs to delete streams for.